### PR TITLE
Added Regex to Recognize Dates

### PR DESCRIPTION
**Pull request by tbell511**
This update improves the quality of the extracted authors because it prevents a date from being extracted as an author.

### CODE REVIEW

File: packages/metascraper-author/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const { getValue, isUrl, titleize } = require('@metascraper/helpers')
 const { isString } = require('lodash')
 
 const REGEX_STRICT = /^\S+\s+\S+/
+const REGEX_DATE = /\d{2,}/g
```
Comment (Kikobeats) [inline packages/metascraper-author/index.js]: Hmmm not sure if global flag is necessary, what do you think?

Comment (formatically) [reply packages/metascraper-author/index.js]: Your right! It's not needed.

File: packages/metascraper-author/index.js [hunk 2]
```diff
@@ -29,7 +30,7 @@ const wrap = rule => ({ htmlDom }) => {
 
 const strict = rule => $ => {
   const value = rule($)
-  return REGEX_STRICT.test(value) && value
+  return REGEX_STRICT.test(value) && !REGEX_DATE.test(value) &&value
```
Comment (Kikobeats) [inline packages/metascraper-author/index.js]: When tests run on travis, first it runs linter:
https://travis-ci.org/microlinkhq/metascraper/jobs/342081136#L3716

It fails beacuse a missing space after `&&` is necessary 🙂

**Comment by Kikobeats** [comment]
Thanks for the PR!

Based on the [original issue](https://github.com/microlinkhq/metascraper/issues/48#issue-286169157) I think you need to move this specific rule:

https://github.com/microlinkhq/metascraper/blob/master/packages/metascraper-author/index.js#L53

before this rule

https://github.com/microlinkhq/metascraper/blob/master/packages/metascraper-author/index.js#L50

In order to apply this improvement correctly.

**Comment by Kikobeats** [comment]
Still waiting the changes @formatically

**Comment by Kikobeats** [comment]
I made the change in the current codebase and it doesn't have any impact; the tests represent the same value.

In order to move this forward, it needs to clarify (adding a new test) why this is necessary.

**Pull request closed without merging**